centrallix/htmlgen/htdrv_datetime.c

```diff
@@ -48,6 +48,13 @@ static struct
     }
     HTDT;
 
+int htdtSetup(pHtSession s)
+	{
+	htrAddStylesheetItem_va(s,"\t.absHidInh { OVERFLOW: hidden; POSITION:absolute; VISIBILITY:inherit;}\n");
```

Comment:
Make sure that all classes begin with the prefix for the widget in question.  In this case "dt".  Also, it doesn't look like this class actually gets applied to any datetime widget DIVs, but it is necessary to replace the original CSS.  Multiple classes can be applied to an HTML element by separating the classes with a space, e.g. `<div class="dtBtn dtAhi"></div>` or similar.

Reply:
I do see this issue in other widgets as well, so I'll go ahead and turn the PR back over to you for updates.